### PR TITLE
C# shift operators: mention unsigned shift where necessary

### PR DESCRIPTION
**Pull request by pkulikov**
Mention the `>>>` operator in relevant places of the [Bitwise and shift operators](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/bitwise-and-shift-operators) article

### CODE REVIEW

File: docs/csharp/language-reference/operators/bitwise-and-shift-operators.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ f1_keywords:
   - "~_CSharpKeyword"
   - "<<_CSharpKeyword"
   - ">>_CSharpKeyword"
+  - ">>>_CSharpKeyword"
```
Comment (pkulikov) [inline docs/csharp/language-reference/operators/bitwise-and-shift-operators.md:10]: This should fix the `F1` navigation from Visual Studio. The compound assignment is already in the list.

File: docs/csharp/language-reference/operators/bitwise-and-shift-operators.md [hunk 2]
```diff
@@ -161,9 +163,9 @@ For the complete list of C# operators ordered by precedence level, see the [Oper
 
 ## Shift count of the shift operators
 
-Prior to C# 11, for the shift operators `<<` and `>>`, the type of the right-hand operand must be `int` or a type that has a [predefined implicit numeric conversion](../builtin-types/numeric-conversions.md#implicit-numeric-conversions) to `int`. This restriction is removed in C# 11.
+For the built-in shift operators `<<`, `>>`, and `>>>`, the type of the right-hand operand must be `int` or a type that has a [predefined implicit numeric conversion](../builtin-types/numeric-conversions.md#implicit-numeric-conversions) to `int`.
```
Comment (pkulikov) [inline docs/csharp/language-reference/operators/bitwise-and-shift-operators.md:166]: As far as I understand, that restriction is removed only for overloaded shift operators. For example, using `long` for the right-hand operand results in error for the built-in shift operator:
image

**Review by BillWagner — APPROVED** [review]
Thanks @pkulikov 

This LGTM, I'll :shipit: now.

**Comment by BillWagner** [comment]
closing and reopening for a fresh build

**Comment by BillWagner** [comment]
closing and reopening for a fresh build

**Pull request merged**